buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersions.kt

```diff
@@ -56,12 +77,28 @@ public class IntellijVersions(intellijVersionsProperties: Properties, overrideBu
       return listOf(versionKey)
     }
 
-    val propertiesList = listOfNotNull(
+    val propertyValue: Any? =
       IntellijVersions::class.memberProperties
         .single { it.name == versionKey }
         .get(this)
-    )
 
-    return propertiesList.flatMap { if (it is List<*>) it else listOf(it) }.map { it as String }
+    if (propertyValue == null) {
```

Comment:
No major change in logic (other than `IllegalStateException` rather than `ClassCastException` in case sth goes awry), just made it clearer